### PR TITLE
calendar, atq, catimg: add Farsi translation

### PR DESCRIPTION
**Pull request by MrMw3**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

File: pages.fa/common/calendar.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+`calendar -A {{30}}`
+
+- نگاهی به گذشته، نمایش رویداد های هفت روز گذشته:
+
+`calendar -B {{7}}`
+
+- نمایش رویداد ها با استفاده از یک فایل تقویم دلخواه:
+
+`calendar -f {{path/to/file}}`
```
Comment (gutjuri) [inline pages.fa/common/calendar.md:20]:
```suggestion
`calendar -f {{مسیر/به/فایل}}`
```

Can we translate `path/to/file` into farsi? Does this make sense in your opinion? (Also, for catimg.md).

**Review by gutjuri — COMMENTED** [review]
Thanks, I have one comment. Could you translate the file paths into Farsi as well? :)

**Review by gutjuri — APPROVED** [review]
LGTM. In my previous feedback, I was not aware of #11997.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**